src/javascripts/ng-admin/es6/lib/DataStore/DataStore.js

```diff
@@ -0,0 +1,50 @@
+
+class DataStore {
+    constructor(name) {
+        this._name = name;
+        this._entry = new Map();
```

Comment:
not a big fan of the double usage. I think you should only keep the `_entries` property and related methods. For views requiring only a single entry, you'd call `store.getFirstEntry(view)`.